main.go

```diff
@@ -29,7 +29,10 @@ var (
 
 func main() {
 	flag.Parse()
-	oidcConfig.Validate()
+	err := oidcConfig.ReadAndValidate()
+	if err != nil {
+		panic(err)
```

Comment:
We should avoid panicking.
In this case due to we are at the main level what I would do is to exit cleanly with an error code, logging the error too.